lib/apple/index.js

```diff
@@ -35,7 +35,12 @@ function parseResult(result) {
 	var receipt = result.receipt;
 
 	return {
+		status: result.status,
```

Comment:
my only comment would be that these additional return values are documented in the Readme.md file